pkg/client/common.go

```diff
@@ -14,6 +16,21 @@ import (
 	"github.com/buildpacks/pack/pkg/logging"
 )
 
+func (c *Client) addManifestToIndex(ctx context.Context, indexRepoName, repoName string, index imgutil.ImageIndex) error {
+	imageRef, err := name.ParseReference(repoName, name.WeakValidation)
+	if err != nil {
+		return fmt.Errorf("'%s' is not a valid manifest reference: %s", style.Symbol(indexRepoName), err)
```

Comment:
```suggestion
		return fmt.Errorf("'%s' is not a valid manifest reference: %s", style.Symbol(repoName), err)
```
I think this is what is meant... in which case `indexRepoName` is unused